### DISCUSSION TITLE
SRT output link problem

### DISCUSSION THREAD

**Discussion by esquilo444** [datarhei/restreamer#674] [General]
Good morning,
Since the last version, the srt link, as well as the use in source for reencoding, has disappeared. Is this normal?
I have to have an already reencoded stream to restream on a platform and I can no longer use the native srt stream to send it back like an srt server...
It's a shame... is there any hope that things will return to the way they were before on this point.
Thank you very much for your work ..
Sincerely.
Jerome

**Comment by ioppermann** [comment]
There's currently a bug in the UI that displays a wrong SRT URL to be pasted into OBS. See this description: https://github.com/datarhei/restreamer/issues/635#issuecomment-1837234576

This bug has been fixed and the fix will be available in the next release.

**Comment by esquilo444** [comment]
Good morning,
Perhaps I expressed myself poorly. I know the bug you told me about. No problem on this point my SRT stream arrives on the server... you actually have to add ".stream" at the end of the ID. My problem concerns the links found under the monitoring window..
There is no longer the direct SRT link which was very useful to me, even essential.. Attached are two photos. My server and yours... Thank you in advance for your insight.
Jerome
![mon serveur](https://github.com/datarhei/restreamer/assets/88659019/8d0fe506-5b41-4c04-a045-836d9d5bca50)
![votre serveur](https://github.com/datarhei/restreamer/assets/88659019/13ea47ed-c14c-4bfd-93b4-6731452c9f6f)

**Comment by ioppermann** [comment]
In order to enable this button, you have to enable the SRT output in the "Processing & Control" section of your channel configuration:
![image](https://github.com/datarhei/restreamer/assets/57445/f34c3425-088e-4b0f-af63-4089269dfa0c)
![image](https://github.com/datarhei/restreamer/assets/57445/71eecda1-4187-4793-b2b6-50bd5348567d)

**Reply by esquilo444** [reply]
Thanks a lot...
You are the best ...
